components/financial_assistance/app/views/financial_assistance/applicants/_edit.html.erb

```diff
@@ -366,54 +366,55 @@
+    $.inputMasks();
+
+      $('#applicant_same_with_primary').click(function(){
+        if($(this).is(':checked')){
+            $('#applicant-home-address-area').css("display", "none");
+        } else {
+            $('#applicant-home-address-area').css("display", "block");
+        }
+      });
 
-    $("#destroyApplicant .modal-cancel-button").click(function(e) {
-      $("#destroyApplicant").modal('hide');
+      $(document).on('click', '#delete_applicant_button', function(e) {
+        e.preventDefault();
+        $("#destroyApplicant").modal();
+
+      $("#destroyApplicant .modal-cancel-button").click(function(e) {
+        $("#destroyApplicant").modal('hide');
+      });
+      $(".btn-confirmation").removeAttr('disabled');
     });
-    $(".btn-confirmation").removeAttr('disabled');
-  });
-
-  function confirmMember(element) {
-    const form = element.closest('form')[0];
-    const maleRadio = document.getElementById('radio_male');
-    const femaleRadio = document.getElementById('radio_female');
-    const confirmButtonText = "<%= l10n('confirm_member') %>";
-    const applicantSSN = document.getElementById('applicant_ssn')
-    const applicantNoSSN = document.getElementById('applicant_no_ssn')
-    element.text("Please Wait...").addClass("disabled");
-
-    PersonValidations.manageRequiredValidations(element);
-
-    if (!form.checkValidity() || !isGenderSelected(maleRadio, femaleRadio) || !isSsnSelected(applicantSSN, applicantNoSSN)) {
-      element.text(confirmButtonText).removeClass("disabled");
+
+    function confirmMember(element) {
+      const form = element.closest('form')[0];
+      const maleRadio = document.getElementById('radio_male');
+      const femaleRadio = document.getElementById('radio_female');
+      const confirmButtonText = "<%= l10n('confirm_member') %>";
+      const applicantSSN = document.getElementById('applicant_ssn')
+      const applicantNoSSN = document.getElementById('applicant_no_ssn')
+      element.text("Please Wait...").addClass("disabled");
+
+      PersonValidations.manageRequiredValidations(element);
+
+      if (!form.checkValidity() || !isGenderSelected(maleRadio, femaleRadio) || !isSsnSelected(applicantSSN, applicantNoSSN)) {
+        element.text(confirmButtonText).removeClass("disabled");
+      }
     }
-  }
 
-  function isGenderSelected(maleRadio, femaleRadio) {
-    return (maleRadio && maleRadio.checked) || (femaleRadio && femaleRadio.checked);
-  }
+    function isGenderSelected(maleRadio, femaleRadio) {
+      return (maleRadio && maleRadio.checked) || (femaleRadio && femaleRadio.checked);
+    }
 
-  function isSsnSelected(applicantSSN, applicantNoSSN) {
-    return (applicantSSN && applicantSSN.value.replace(/-/g, '').length == 9) || (applicantNoSSN && applicantNoSSN.checked);
-  }
+    function isSsnSelected(applicantSSN, applicantNoSSN) {
+      return (applicantSSN && applicantSSN.value.replace(/-/g, '').length == 9) || (applicantNoSSN && applicantNoSSN.checked);
+    }
```

Comment:
Given our issues with render of rjs->erb->missing nonce render chain, we want to move these event handlers out of here and into the JS assets.